app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -191,7 +191,11 @@ def handle_failed_status(enrollment, response)
     )
 
     enrollment.update(status: :failed)
-    send_failed_email(enrollment.user, enrollment)
+    if response['fraudSuspected'] == 'true'
```

Comment:
Unless you verified this by testing or reviewing the API output, I would recommend hedging our bets by using `to_s` here.

```suggestion
    if response['fraudSuspected']&.to_s == 'true'
```

This would convert a `false` or `true` boolean into the corresponding string `'false'` or `'true'`. I missing value would be `nil`.

Reply:
@sheldon-b had done some related testing at https://github.com/18F/identity-idp/pull/6967#discussion_r977613182 and discovered these are returned by USPS as booleans, not strings (despite what their documentation states). `to_s` could be a safe way to handle both (though I'd hope we could rely on a single known value type).

Reply:
Thanks, that works for me

Reply:
i'm going to change this to a boolean given this info